docs/Homebrew-on-Linux.md

```diff
@@ -15,16 +15,17 @@ The Homebrew package manager may be used on Linux and [Windows Subsystem for Lin
 
 ## Features
 
-- Can install software to your home directory and so does not require *sudo*
 - Install software not packaged by your host distribution
 - Install up-to-date versions of software when your host distribution is old
 - Use the same package manager to manage your macOS, Linux, and Windows systems
 
 ## Install
 
-Instructions for a supported install of Homebrew on Linux are on the [homepage](https://brew.sh).
+Instructions for  the best, supported install of Homebrew on Linux are on the [homepage](https://brew.sh).
 
-The installation script installs Homebrew to `/home/linuxbrew/.linuxbrew` using *sudo* if possible and within your home directory at `~/.linuxbrew` otherwise. Homebrew does not use *sudo* after installation. Using `/home/linuxbrew/.linuxbrew` allows the use of more binary packages (bottles) than installing in your personal home directory.
+The installation script installs Homebrew to `/home/linuxbrew/.linuxbrew` using *sudo* if possible and within your home directory at `~/.linuxbrew` otherwise. Homebrew does not use *sudo* after installation. Using `/home/linuxbrew/.linuxbrew` allows the use of most binary packages (bottles) which will not work when installing e.g. in your personal home directory.
```

Comment:
```suggestion
The installation script installs Homebrew to `/home/linuxbrew/.linuxbrew` using *sudo* if possible and within your home directory at `~/.linuxbrew` otherwise. Homebrew does not use *sudo* after installation. Using `/home/linuxbrew/.linuxbrew` allows the use of most binary packages (bottles) which will not work when installing in e.g. your personal home directory.
```